app/screens/channel_bookmark/index.tsx

```diff
@@ -269,6 +270,7 @@ const ChannelBookmarkAddOrEdit = ({
             edges={edges}
             style={styles.content}
             testID='channel_bookmark.screen'
+            nativeID={SecurityManager.getShieldScreenId(componentId)}
```

Comment:
This seems something we will need to add for every screen we create from now on. Is there any way to enforce it (by linting rule, for example)?

Or what about to add a wrapper directly on the screen builder like we have `withDatabase` and so on?

Reply:
the thing is that not necessarily this is applied in the screens folder but it should be somewhere in the component tree of a screen, although the changes in the emm library have a fallback in case none is found.

Also, a HOC may not be what we want cause is not always the first "View" that has this id